pkg/api/defaults-kubelet_test.go

```diff
@@ -87,10 +88,11 @@ func TestKubeletConfigDefaults(t *testing.T) {
 	expected["--resolv-conf"] = "\"\"\"\""
 	expected["--eviction-hard"] = "\"\"\"\""
 	delete(expected, "--pod-manifest-path")
+	delete(expected, "--protect-kernel-defaults")
```

Comment:
and @andyzhangx this is the unit test that validates that `--protect-kernel-defaults` is not present as a kubelet flag for Windows nodes